diff --git a/node_modules/pdf-parse/index.js b/node_modules/pdf-parse/index.js
index b288c42..2c14044 100644
--- a/node_modules/pdf-parse/index.js
+++ b/node_modules/pdf-parse/index.js
@@ -4,7 +4,7 @@
 const util = require('util');
 const path = require('path');
 const fs = require('fs');
-fs.readFileSync(path.join(__dirname, '../test/data/05-versions-space.pdf'));
+//fs.readFileSync(path.join(__dirname, '../test/data/05-versions-space.pdf'));
 
 
 const PDF_INFO_MAP = {
