diff --git a/node_modules/pdf-parse/lib/pdf-parse.js b/node_modules/pdf-parse/lib/pdf-parse.js
index 0375a6c..3e6027a 100644
--- a/node_modules/pdf-parse/lib/pdf-parse.js
+++ b/node_modules/pdf-parse/lib/pdf-parse.js
@@ -5,7 +5,6 @@
 
 //see https://github.com/mozilla/pdf.js/blob/628e70fbb5dea3b9066aa2d0b323a443510c3713/src/shared/util.js#L683
 const pdfjs = require('../pdf.js/build/pdf.js');
-const PDF_DATA = new Uint8Array(fs.readFileSync('./test/data/05-versions-space.pdf'));
 
 function render_page(pageData) {
     //check documents https://mozilla.github.io/pdf.js/